### PR TITLE
Add guide on creating multi-arch manifests

### PR DESCRIPTION
**Pull request by Jasper-Ben**
While kaniko itself does not natively support building multi-arch container manifests, it may be used in combination with tools such as manifest-tool to create and merge seperate arch builds into a single manifest.

Fixes https://github.com/GoogleContainerTools/kaniko/issues/1102 
Fixes https://github.com/GoogleContainerTools/kaniko/issues/786

**Description**

While kaniko itself does not natively support building multi-arch container manifests, it may be used in combination with tools such as manifest-tool to create and merge seperate arch builds into a single manifest.

**Submitter Checklist**

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [unit tests](../DEVELOPMENT.md#creating-a-pr)
- [x] Adds integration tests if needed.

_See [the contribution guide](../CONTRIBUTING.md) for more details._

**Reviewer Notes**

- [ ] The code flow looks good. 
- [ ] Unit tests and or integration tests added.

### CODE REVIEW

**Comment by jerroldgao** [comment]
I personally do not favor this, your manifest tool is over 200MB by size, if so, why not use a docker image which is only 100MB unless you are in windows?

**Comment by aaron-prindle** [comment]
@Jasper-Ben thanks for the PR here!   If you can rebase this, I'd be happy to review and get this in!

**Comment by eugenejen** [comment]
@Jasper-Ben do you need help on this PR to rebased it? I have vested interest to enable kaniko multiarch build so i will like to help.

**Comment by Jasper-Ben** [comment]
@aaron-prindle @eugenejen 
Hi! Sorry for the late reply, just didn't get around to it yet. Will rebase now :slightly_smiling_face:

**Comment by Jasper-Ben** [comment]
Done.

I also noticed a missing toc entry, which I fixed in separate commit https://github.com/GoogleContainerTools/kaniko/pull/2306/commits/b097cd2b95d5cfc3f06fce39c0501b32a9d6d533

Feel free to cherry-pick into separate PR, if you so prefer :slightly_smiling_face:

**Review by eugenejen — APPROVED** [review]
LRTM

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ _If you are interested in contributing to kaniko, see
       - [Flag `--oci-layout-path`](#flag---oci-layout-path)
       - [Flag `--push-retry`](#flag---push-retry)
       - [Flag `--registry-certificate`](#flag---registry-certificate)
+      - [Flag `--registry-certificate`](#flag---registry-client-cert)
```
[the remark this answers is not in the document]
Comment (Jasper-Ben) [reply README.md:97]: Maybe. It has a section in the README though:

https://github.com/GoogleContainerTools/kaniko/#flag---registry-client-cert

So either the TOC is wrong or the section shouldn't exist in the first place?

[the remark this answers is not in the document]
Comment (eugenejen) [reply README.md:97]: i meant the newline has the same markdown --registry-certificate as the line above. but the flag value is --registry-client-cert. 

i just wondered why the markdown is dupe and inconsistent to the flag value here.

**Comment by Jasper-Ben** [comment]
Good catch! Thats on me. Doctoc kept changing the document layout for me, so I changed this part manually. Will push a fix tomorrow.

**Comment by aaron-prindle** [comment]
Thanks for updating the PR here @Jasper-Ben!  Once the last fix you mentioned is in we can merge 🎊

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jasper-Ben) [reply README.md]: fixed :slightly_smiling_face:

**Review by eugenejen — APPROVED** [review]

**Comment by eugenejen** [comment]
@aaron-prindle can you look at the PR?

**Review by aaron-prindle — APPROVED** [review]
LGTM!  Thanks for this @Jasper-Ben!

**Pull request merged**